browser/main/TopBar/index.js

```diff
@@ -244,6 +250,14 @@ class TopBar extends React.Component {
     })
   }
 
+  handleFocusSearch () {
```

Comment:
Actually, there is a method named `handleSearchFocus()`... Thus it's a little confusing :sob: